src/main/java/entity/ArticleEntity.java

```diff
@@ -0,0 +1,127 @@
+package entity;
+import javax.persistence.*;
+import java.sql.Timestamp;
+import java.util.Objects;
+import java.util.Set;
+import java.util.UUID;
+
+@Entity
+@Table(name="ARTICLE")
+public class ArticleEntity {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private UUID id;
+
+    @Column(name = "TITLE", length = 255, nullable = false, unique = false)
+    private String title ;
+    @Column(name = "TEXT", length = 4000, nullable = false, unique = false)
```

Comment:
I'm not sure if 4000 symbols are enough. Let's mark it:
@Column(columnDefinition="TEXT")